packages/core/sdk/src/internal/common/mongodb-extjson.d.ts

```diff
@@ -15,18 +15,18 @@
  */
```

Comment:
Some interesting changes on this file, it seems auto-fix did the same thing as above here with the type / interface

Reply:
Looking online, it seems like it it better to use interfaces as opposed to types which seem to just be aliases